ports/bfgroup-lyra/portfile.cmake

```
@@ -1,21 +1,24 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO bfgroup/Lyra
-    REF 1.6
-    SHA512 e357fd0e925b67a51ad7232035ac66842676837baebf7a69eb416807b11400c283d098a22bf3ae27ce904700c5b849953ede1873d6535a8b34c4704ebcb09748
-    HEAD_REF master
+    REF "${VERSION}"
+    SHA512 643c25fbe996af2e888eacb99a715e3d420dbfc21d48756703cf301ab6ba0d1f8eea1cd0764bd5c173d2ddcef7c799448d8c3a77676024205163305e1363d461
+    HEAD_REF release

```

```suggestion
    HEAD_REF develop
```
The default upstream branch is develop.

---

Yes, but we use release branch here. And 1.6.1 tag is on release branch.